serializers.go

```diff
@@ -15,6 +15,17 @@ import (
 )
 
 func (p OnlineQueryParams) serialize() (*internal.OnlineQueryRequestSerialized, error) {
+	if len(p.outputs) > 1 && p.QueryName != "" {
+		return nil, errors.New("please specify either outputs or query name, not both")
+	}
```

Comment:
this breaks compatibility for existing users - we could warn, but we should not error here